src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -1300,6 +1355,13 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
           {confirmCopy}
         </Button>
       </CardFooter>
+      <FeeModal
+        affiliateFeeAmountUserCurrency={confirmedQuote?.feeAmountFiat ?? '0'}
+        isOpen={showFeeModal}
+        onClose={toggleFeeModal}
+        inputAmountUsd={totalAmountFiat}
```

Comment:
fiat !== usd

Reply:
845994b0c34bbb8cd6ca27eccdfef204eca4f014, 685da7e2df37837715dc607c52a43d821fa6f6a0, f8f5855e97fd19e1f88f5dd6d8c37b035b38f8d4 and 928142ed51699c8fd031247b8c95b28d6fe73712

While at it:

- Also removes the duplicate "Fees" row (effectively the same as "ShapeShift fee"
- Ensures the fee is displayed as the correct user-currency fee when using a currency other than USD

Reply:
🐐